macros/create_artifact_resources.sql

```diff
@@ -9,21 +16,132 @@ create stage if not exists {{ src_dbt_artifacts }}
 file_format = (type = json);
 {% endset %}
 
+{% set create_new_stage_query %}
+create stage if not exists {{ artifact_stage }}
+file_format = ( type =  json );
+{% endset %}
+
 {% set create_table_query %}
 create table if not exists {{ src_dbt_artifacts }} (
     data variant,
     generated_at timestamp,
     path string,
     artifact_type string
 );
+{% endset %}
+
+{% set create_results_query %}
+create table if not exists {{ src_results }} (
+    command_invocation_id string,
+    dbt_cloud_run_id int,
+    artifact_run_id string,
+    artifact_generated_at timestamp_ntz,
```

Comment:
Looks like all dbt artifact timestamps are in UTC: https://github.com/brooklyn-data/dbt_artifacts/blob/main/integration_test_project/artifacts/schema_v4/build/run_results.json#L5
```suggestion
    artifact_generated_at timestamp_tz,
```